### PR TITLE
[CONTAINS MIGRATION][DO-2198] Save data with unidentified PEP and Company

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: business_register/converter/declaration.py [hunk 1]
```diff
@@ -792,65 +807,63 @@ def save_bank_account(self, declaration_data, declaration, pep):
...
         for data in account_data:
+            bank = self.UNIDENTIFIED_COMPANY
+            registration_number = ''
             bank_code_ua = data.get('establishment_ua_company_code')
             if bank_code_ua in self.NO_DATA:
                 bank_code_ua = ''
             bank_code_en = data.get('establishment_eng_company_code')
             if bank_code_en in self.NO_DATA:
                 bank_code_en = ''
-            if not bank_code_en and not bank_code_ua or bank_code_en in all_banks or bank_code_ua in all_banks:
```
Comment (EmmaSamoilova) [inline business_register/converter/declaration.py:801]: Why did you delete this? It seems to be agreed that banks should not be repeated

File: business_register/converter/declaration.py [hunk 2]
```diff
@@ -1049,10 +1050,7 @@ def save_income(self, declaration_data, declaration):
         for data in incomes_data:
             income_type = data.get('objectType')
```
Comment (EmmaSamoilova) [inline business_register/converter/declaration.py:1052]: Maybe should delete this line too?

File: business_register/models/declaration_models.py [hunk 3]
```diff
@@ -736,6 +738,7 @@ class Income(DataOceanModel):
         (SALE_OF_MOVABLES, 'Sale of movables'),
         (SELF_EMPLOYMENT, 'Self-employment'),
         (ROYALTY, 'Royalty'),
+        (NO_INFO_FROM_FAMILY_MEMBER, 'Family member did not provide the information'),
```
Comment (EmmaSamoilova) [inline business_register/models/declaration_models.py:741]: You added new type NO_INFO_FROM_FAMILY_MEMBER, but didn't added this in save_income to types

Comment (IegorSoboliev) [reply business_register/models/declaration_models.py:741]: Thanks!

File: business_register/converter/declaration.py (hunk 1 above)
Comment (IegorSoboliev) [reply business_register/converter/declaration.py:801]: I see. Restored

File: business_register/converter/declaration.py (hunk 2 above)
Comment (IegorSoboliev) [reply business_register/converter/declaration.py]: Thanks! Deleted with TODO)

**Review by EmmaSamoilova — APPROVED** [review]

File: business_register/converter/declaration.py [hunk 4]
```diff
@@ -79,12 +79,22 @@ def __init__(self):
...
         self.UKRAINE = Country.objects.get(name='ukraine')
-        self.keys = set()
+        self.UNIDENTIFIED_PEP, _ = Pep.include_deleted_objects.get_or_create(
+            first_name='',
+            middle_name='',
+            last_name='UNIDENTIFIED_PEP',
+            fullname='UNIDENTIFIED_PEP',
+            is_dead=True,
+            deleted_at='2021-01-01 10:05:26.268544'
+        )
```
Comment (roman-tiukh) [inline business_register/converter/declaration.py:92]: `'UNIDENTIFIED_PEP'` - should be constant
```suggestion
        self.UNIDENTIFIED_PEP = Pep.get_unidentified_pep()
```

File: business_register/converter/declaration.py [hunk 5]
```diff
@@ -79,12 +79,22 @@ def __init__(self):
...
+        self.UNIDENTIFIED_COMPANY, _ = Company.include_deleted_objects.get_or_create(
+            name='UNIDENTIFIED_COMPANY',
+            code='666',
+            deleted_at='2021-01-01 10:05:26.268544'
+        )
```
Comment (roman-tiukh) [inline business_register/converter/declaration.py:97]: `'UNIDENTIFIED_COMPANY'` - should be constant
```suggestion
        self.UNIDENTIFIED_COMPANY = Company.get_unidentified_company()
```

File: business_register/converter/declaration.py [hunk 6]
```diff
@@ -747,41 +765,40 @@ def save_liability(self, declaration_data, declaration):
                 if owner_info:
                     owner_id = owner_info[0].get('person')
             if owner_id not in self.NO_DATA:
-                if owner_id == '1':
+                if owner_id == self.DECLARANT:
```
Comment (roman-tiukh) [inline business_register/converter/declaration.py:768]: :+1:

File: business_register/models/declaration_models.py [hunk 7]
```diff
@@ -775,6 +778,13 @@ class Income(DataOceanModel):
         verbose_name='paid by',
         help_text='company or organisation that paid'
     )
+    company_registration_number = models.CharField(
+        'registration number',
```
Comment (roman-tiukh) [inline business_register/models/declaration_models.py:782]: Minor:
redundant verbose_name, it duplicate info in field name

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/converter/declaration.py (hunk 4 above)
Comment (IegorSoboliev) [reply business_register/converter/declaration.py:92]: done

File: business_register/converter/declaration.py (hunk 5 above)
Comment (IegorSoboliev) [reply business_register/converter/declaration.py:97]: done

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**